cleanup_switzerland/models/database_cleanup.py

```diff
@@ -77,62 +79,69 @@ def clean_mail_message(self):
         today = date.today()
         limit = today - relativedelta(years=1)
         limit_str = limit.strftime("%Y-%m-%d")
-        self.env.cr.execute(
-            "DELETE FROM mail_message WHERE model = 'account.invoice' "
-            "AND res_id IN (SELECT id FROM account_invoice WHERE state IN ("
-            "'cancel', 'paid') AND date_invoice < %s LIMIT 1000)", [limit_str]
-        )
-        count = self.env.cr.rowcount
-        self.env.cr.commit()  # pylint: disable=invalid-commit
-        _logger.info('{} old invoice messages removed'.format(count))
 
-        self.env.cr.execute(
-            "DELETE FROM mail_message WHERE model='gmc.message.pool' AND "
-            "res_id IN (SELECT id FROM gmc_message_pool WHERE state ="
-            "'success' LIMIT 1000)"
-        )
-        count = self.env.cr.rowcount
-        self.env.cr.commit()  # pylint: disable=invalid-commit
-        _logger.info('{} old gmc_pool messages removed'.format(count))
+        if 'account.invoice' in self.env:
+            self.env.cr.execute(
+                "DELETE FROM mail_message WHERE model = 'account.invoice' "
+                "AND res_id IN (SELECT id FROM account_invoice WHERE state IN ("
+                "'cancel', 'paid') AND date_invoice < %s LIMIT 1000)", [limit_str]
+            )
+            count = self.env.cr.rowcount
+            self.env.cr.commit()  # pylint: disable=invalid-commit
+            _logger.info('{} old invoice messages removed'.format(count))
 
-        self.env.cr.execute(
-            "DELETE FROM mail_message WHERE model='partner.communication.job' "
-            "AND res_id IN (SELECT id FROM partner_communication_job WHERE "
-            "state = 'cancel' OR (state = 'done' AND sent_date < %s)"
-            "LIMIT 1000)",
-            [limit_str]
-        )
-        count = self.env.cr.rowcount
-        self.env.cr.commit()  # pylint: disable=invalid-commit
-        _logger.info('{} old partner_communication messages removed'.format(
-            count))
+        if 'gmc.message.pool' in self.env:
```

Comment:
I added these if-conditions so I didn't need to add all dependencies in the manifest.